Calculator/Calculator/Program.cs

```diff
@@ -0,0 +1,105 @@
+using System;
+
+namespace Calculator
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            int previous = default;
+            int current = default;
+            string operation = default;
+            
+            Console.WriteLine("Калькулятор поддерживает только такие операции: + - * / ^ !");
+
+            while(true)
+            {
+                var askFirstOperand = true;
+                var askSecondOperand = true;
+                var askOperation = true;
+                
+                while(askFirstOperand)
+                {
+                    Console.Write("Введите число: ");
+                    var priv = Console.ReadLine();
```

Comment:
maybe then `firstOperand` instead of `priv`, sounds like a privet))))

Reply:
Yeah, I think it would be better